client/src/components/specs/pageSpec.js

```diff
@@ -10,4 +11,20 @@ describe('<Page />', () => {
 
     expect(tree).toMatchSnapshot();
   });
+
+  test('Renders correct state based on passed in values', () => {
+    const wrapper = shallow(<Page show={false} isFollowing={true} hello={false}/>);
```

Comment:
Maybe you could pull this out into variables?
```suggestion
    const wrapper = shallow(<Page show={false} isFollowing={true} hello={false}/>);
    const instance = wrapper.instance();
    const { state } = instance;
```

Reply:
great suggestion! added